assets/js/theme/common/carousel.js

```diff
@@ -14,7 +24,24 @@ export default function () {
 
     if ($carousel.length === 0) return;
 
-    $carousel.slick({ dots: $carousel[0].childElementCount > 1 });
+    const isMultipleSlides = $carousel[0].childElementCount > 1;
+
+    const slickSettingsObj = isMultipleSlides
+        ? {
+            dots: true,
+            customPaging: () => (
```

Comment:
Why do we need to return empty button here?

Reply:
For current ticket it is not absolutely necessary, because slick will generate it by default. But I still added it manually for 2 reasons: to make obvious that we have buttons here, and the main - in the next implementations we need to set aria-labels for this generated dot buttons, so we will need this function for it. But if you insist I can remove customPaging function for now

Reply:
It's fine, thank you for clarification